src/ImageSharp/Formats/WebP/LossyUtils.cs

```diff
@@ -0,0 +1,912 @@
+// Copyright (c) Six Labors and contributors.
+// Licensed under the Apache License, Version 2.0.
+
+using System;
+using System.Buffers.Binary;
+
+namespace SixLabors.ImageSharp.Formats.WebP
+{
+    internal static class LossyUtils
+    {
+        private static void Put16(int v, Span<byte> dst)
+        {
+            for (int j = 0; j < 16; ++j)
+            {
+                Memset(dst.Slice(j * WebPConstants.Bps), (byte)v, 0, 16);
+            }
+        }
+
+        public static void DC16(Span<byte> dst, Span<byte> yuv, int offset)
+        {
```

Comment:
Define offsetMinus1 = offset - 1 before the loop, use that in the first yuv[] index.
Define int offsetMinusBps = offset - WebPConstants.Bps before the loop, use that as part of the second yuv[] index.

By that we'd get:
```C#
int offsetMinus1 = offset - 1;
int offsetMinusBps = offset - WebPConstants.Bps;

int dc = 16;
for (int j=0; j<16; ++j) 
{
  dc += yuv[offsetMinus1 + (j * WebPConstants.Bps) + yuv[offsetMinusBps + j];
}

Put16(dc >> 5, dst);
```